dg.go

```diff
@@ -23,21 +24,34 @@ type directedGraph[NodeType any] struct {
     connectionsToNode   map[string]map[string]struct{}
 }
 
+var errorPathRegex, _ = regexp.Compile("error|crashed|failed|deploy_failed")
+
 func (d *directedGraph[NodeType]) Mermaid() string {
```

Comment:
Is there any testing of `Mermaid()`?  If not, should there be?  (If there were, it could confirm that the `Compile()` call returns no error, rather than just ignoring it....)